src/Panel/Panel.php

```diff
@@ -228,10 +228,11 @@ public static function customJs(App $kirby)
     public static function data(App $kirby, string $component, array $data = []): array
     {
         // shared default data
+        $multilang = $kirby->option('languages', false) !== false;
```

Comment:
If I haven't missed anything, why don't we use the `$kirby->multilang()` method? 🤔

Reply:
I think the definition is a bit different: `$kirby->multilang()` checks if there actually has been at least one language defined. While we here are checking if the config allows defining languages - no matter if the first one has been defined or not.

Reply:
Ah right 👍